ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -50,6 +50,7 @@ export enum TransactionAuthenticatorVariant {
   TransactionAuthenticatorEd25519 = 0,
   TransactionAuthenticatorMultiEd25519 = 1,
   TransactionAuthenticatorMultiAgent = 2,
+  TransactionAuthenticatorSecp256k1 = 3,
   TransactionAuthenticatorFeePayer = 4,
```

Comment:
```suggestion
  TransactionAuthenticatorFeePayer = 3,
  TransactionAuthenticatorSecp256k1 = 4,
```

The enum variants will always match the Rust enum, so make sure to check the *.rs file if ever in doubt